app/admin/impressions.rb

```diff
@@ -44,10 +44,15 @@ def pie(data, title, id)
 			end
 		end
 		panel "Pup Statistics" do
+			repeated_params = {library: {animation: {duration: 1050, easing: 'easeOutQuad' }},
```

Comment:
Is this a necessary change? repeated_params doesn't seem to be reused any where else.  Am I missing what it's doing?

Reply:
Oops I just did that to appease the codeclimate for DRY. Will try to reuse for the other function calls